src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD = 'Bookstore/books/ADD';
+const REMOVE = 'Bookstore/books/REMOVE';
```

Comment:
- [ ] [OPTIONAL] Just to make it perfect and follow [ducks pattern](https://github.com/erikras/ducks-modular-redux), You can name them like `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Reply:
Thanks, I'm changing it right now. 👍